Lesson16/feyza/script.js

```diff
@@ -0,0 +1,250 @@
+
+function isPositive(number){
+	if(number > 0) {
+		return "positive";
+	} else if(number === 0){ 
+	    return "zero";
+	} else{
+        return "negative";
+	}
+}
+console.log("Exercise 5", isPositive(29));
+
+// 6. Write a function that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400)
+// or not using if else statement.
+
+function isYears(leapYear){
+	if(leapYear % 4 === 0 && leapYear % 100 !== 0 && leapYear % 400 !== 0){
+		return "It is a leap year";
+	} else{
+		return "It is not a leap year";
+	}
+}
+console.log("Exercise 6", isYears(2035));
+
+// 7. Write a function that takes in a temperature and checks if it's above or below freezing using if else statement.
+
+function isTemperature(degree){
+	if(degree > 0){
+		return "Temperature is above freezing";
+	}else if(degree < 0){
+		return "Temparature is below freezing";
+	} 
+}
+console.log("Exercise 7",isTemperature(-18));
+
+// 8. Write a function that takes in a person's height and weight and checks if they are considered underweight, normal, overweight, or obese using if else statement.
+// Body Mass Index (BMI) is a person’s weight in kilograms (or pounds) divided by the square of height in meters (or feet). A high BMI can indicate high body fatness.
+//Formula: weight (kg) / [height (m)]2
+// Below 18.5	Underweight
+// 18.5 – 24.9	Healthy Weight
+// 25.0 – 29.9	Overweight
+// 30.0 and Above	Obesity
+
+
+function BMI(height,weight){
+let result = weight / height * 2;
+   if(result < 18.5){
+	console.log("Exercise 8", "Underweight", + result);
+   } else if(result >= 18.5 && result <= 24.9){
+	console.log("Exercise 8", "Healty Weight", + result);
+   } else if(result >= 25 && result <= 29.9){
+	console.log("Exercise 8", "Overweight", + result);
+   } else if(result >= 30){
+	console.log("Exercise 8", "Obesity", + result);
+   }
+}
+console.log(BMI(55,1.58));
+
+// 9. Write a function that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64),
+// or senior citizen (age 65 and above) using if else statement.
+
+function isPersonAge(age){
+	if(age > 0 && age <= 12){  
+	return "Child";
+} else if (age >= 13 && age <= 19){
+    return "Teenager";
+} else if(age >= 20 && age <= 64){
+	return "Adult";
+} else if(age >= 65){
+    return "Senior Citizen";
+}
+}
+console.log("Exercise 9", isPersonAge(28));
+
+// 10. Write a function that takes age as a number and whether they are a student (yes or no). If the age is less than 18 and the user is a student, 
+//print "You are a student and not old enough to vote". If the age is between 18 and 65 (inclusive) and 
+//the user is not a student, print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen".
+// If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
+
+
+function canVote(age){
+	let person = student;
+	if(age < 18 && person){
```

Comment:
After you update the function definition to take to parameters, you can use second parameter here and below instead of `person` variable. Then the fucntion should work correctly.